language_nim.lua

```diff
@@ -0,0 +1,11 @@
+local config = require "core.config"
+local linter = require "plugins.linter"
+
+config.nimcheck_args = {}
+
+linter.add_language {
+  file_patterns = {"%.nim$"},
+  warning_pattern = "[^:]%((%d+), (%d+)%) ([^\n]+)",
+  command = "nim check $ARGS $FILENAME 2>&1",
+  args = config.luacheck_args
```

Comment:
You created the nimcheck_args variable in the configuration but passed the config.luacheck_args when adding the language.

Reply:
Not only that, but also the wrong file name. Which I copied and destroyed my nim syntax highlighting work which I did 2 hours(